### DISCUSSION TITLE
https://theme-hope.vuejs.press/guide/layout/page.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#877] [Announcements]
# https://theme-hope.vuejs.press/guide/layout/page.html

Icon Support You can configure the icon field in the frontmatter of the page, and fill in the FontClass of the corresponding icon to bind the icon to the page. This icon is used...

https://theme-hope.vuejs.press/guide/layout/page.html

**Comment by alexeyzimarev** [comment]
> In desktop mode, a list of article headers will automatically be displayed on the right side of the screen. 

I don't think this is correct as the ToC is shown as "On This Page" widget. My question is if the ToC can indeed be shown on the right side of the screen as "On This Page" seems counter-intuitive.

**Reply by Mister-Hope** [reply]
Why the title is counter-intuitive?

Anyway, you can choose to hide this title by sending an empty string to i18n configuration

**Reply by alexeyzimarev** [reply]
It's not about the title. I found out that the headers are indeed shown on the right side of the page when there's enough width. Otherwise, the ToC is collapsed to "On this page" widget.

**Reply by Mister-Hope** [reply]
I understand your needs now, if you want to switch to PC layout on lower screen width, you can modify css breakpoints in `.vuepress/styles/config.scss`

**Comment by redxXxkiller** [comment]
How to make the "On This Page" sidebar on the right?

**Reply by Mister-Hope** [reply]
https://github.com/vuepress-theme-hope/giscus-discussions/discussions/877#discussioncomment-12811055
